requirements.txt

```diff
@@ -4,3 +4,4 @@ qdarkstyle==2.8
 peewee==3.13.1
 voluptuous==0.11.7
 pytz
+toml
```

Comment:
pypi is still using version 0.10.0 while the latest develop as of this PR is 0.10.1.